src/formatDate.js

```diff
@@ -48,9 +48,56 @@
  *
  * @returns {string}
  */
+function newDateYear(item) {
+  if (+item < 30) {
+    return 20;
+  } else {
+    return 19;
+  }
+}
+
+function newDateYearIndex(value) {
+  if (value.length === 4) {
+    return value === 'YYYY';
+  } else {
+    return value === 'YY';
+  }
+}
+
+function newDateYearIndexM() {
+  return 'MM';
+}
```

Comment:
I don't see a reason why you need this function at all. You can use just 'MM' value for it, or some constant...